data-plane/pom.xml

```diff
@@ -113,127 +114,55 @@
         <enabled>true</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>red-hat-ga</id>
+      <url>https://maven.repository.redhat.com/ga</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
   </repositories>
 
   <dependencyManagement>
     <dependencies>
       <!-- Vertx -->
       <dependency>
-        <groupId>io.quarkus</groupId>
+        <groupId>com.redhat.quarkus.platform</groupId>
         <artifactId>quarkus-bom</artifactId>
         <version>${quarkus.version}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
 
       <dependency>
-        <groupId>org.antlr</groupId>
-        <artifactId>antlr4-runtime</artifactId>
-        <version>${antlr.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.antlr</groupId>
-        <artifactId>antlr4</artifactId>
-        <version>${antlr.version}</version>
-      </dependency>
-
-      <dependency>
-        <groupId>io.vertx</groupId>
-        <artifactId>vertx-opentelemetry</artifactId>
-        <version>${vertx.version}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>io.vertx</groupId>
-            <artifactId>vertx-core</artifactId>
-          </exclusion>
-        </exclusions>
-      </dependency>
```

Comment:
I don't think this is part of Quarkus

Reply:
You're probably right wrt/ upstream Quarkus vs. RHBQ that included "extended" platform participants, or whatever it's called. I'll check that.